mobile/ios/scripts/build_arch.sh

```diff
@@ -13,10 +13,17 @@ if [ "$PLATFORM" = "iPhoneOS" ]; then
     EXTRA_CONFIG="--host=arm-apple-darwin14 --target=arm-apple-darwin14"
 fi
 
+ROOTDIR=$(cd $(dirname $0) && pwd -P)
+SOURCEDIR="$ROOTDIR/../../../"
+REPOROOT="$ROOTDIR/../"
+
+AVAIL_SDKS=`xcodebuild -showsdks | grep "iphoneos"`
```

Comment:
What if this variable is empty (i.e. one has not installed the iphoneos SDK but maybe only the emulator)?